src/three/math/OBB.js

```diff
@@ -13,9 +14,10 @@ export class OBB {
 
 	update() {
 
-		const { points, inverseTransform, transform, box } = this;
+		const { points, inverseTransform, transform, box, orientedBox } = this;
 		inverseTransform.copy( transform ).invert();
 
+		orientedBox.copy( box ).applyMatrix4( transform );
```

Comment:
If you transform the box by the transform you just get an enlarged AABB. We'll want to store the planes of the OBB (or generate them on the fly) so we can use those to test the frustum points.